source/containers.tex

```diff
@@ -9222,7 +9222,6 @@
 
     // bit reference
     class @\libmember{reference}{vector<bool>}@ {
-      friend class vector;
       constexpr reference() noexcept;
```

Comment:
This private constructor declaration is probably observable because of something about SFINAE/overload-resolution. But I doubt that its `constexpr` and `noexcept` specifiers are observable.

(Similarly below, the fact that the copy ctor is `=default`ed is unobservable, but perhaps it should be marked `noexcept`. Right now its noexceptness depends on the nothrow-copyability of this class's unseen data members.)

Same comments on `bitset::reference` below.

Reply:
So?  This pull request is not about the private default constructor.

Reply:
> This private constructor declaration is probably observable because of something about SFINAE/overload-resolution. But I doubt that its `constexpr` and `noexcept` specifiers are observable.

Agree. LWG3160 depicted how can its existence be observed.

Currently, LWG4140 is dropping the useless default ctor `bitset<N>::reference`, but still leaves `vector<bool>::reference` untouched. @jwakely should we just extend LWG4140 to modify `vector<bool>::reference` or create a new LWG issue?

Reply:
I will add it to the issue, thanks for pointing it out.